CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-﻿3.0 (unreleased)
+﻿3.0.0 (unreleased)
 =================
```

Comment:
@sjhannon Now I'm actually not sure what the convention is or even if there's one. There are version numbers like `1.0` in this file. So it may be that the milestone needs fixing or that the bot needs to do better comparison. Sorry about the confusion.
@bsipocz Can you comment on this?